lib/sinon/util/fake_xml_http_request.js

```diff
@@ -54,22 +54,22 @@ sinonXhr.supportsCORS = isReactNative ||
 var unsafeHeaders = {
     "Accept-Charset": true,
     "Accept-Encoding": true,
-    Connection: true,
+    "Connection": true,
```

Comment:
I think it is just to be consistent - a visual thing. Some people object to using quotes for just some of the keys. Not seeing a big problem :-)